.github/actions/prepare-uberjar-artifact/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Prepare Uberjar Artifact
+runs:
+  using: "composite"
+  steps:
+    - name: Mark with the commit hash
+      run: git rev-parse --short HEAD > COMMIT-ID
+    - name: Calculate SHA256 checksum
+      run: sha256sum ./target/uberjar/metabase.jar > SHA256.sum
```

Comment:
this needs `shell` property, otherwise it'll fail (see the composite action docs, or `prepare-backend` for an example).